src/rp2_common/pico_cyw43_driver/CMakeLists.txt

```diff
@@ -155,4 +155,104 @@ if (EXISTS ${PICO_CYW43_DRIVER_PATH}/${CYW43_DRIVER_TEST_FILE})
     endfunction()
 
     pico_promote_common_scope_vars()
+
+    # Set env var, so it can be accessed in the function
+    set(ENV{PICO_CYW43_DRIVER_CURRENT_PATH} ${CMAKE_CURRENT_LIST_DIR})
+    function(pico_use_partition_firmware TARGET)
+        target_compile_definitions(${TARGET} PRIVATE CYW43_USE_PARTITION_FIRMWARE=1)
+        pico_embed_pt_in_binary(${TARGET} $ENV{PICO_CYW43_DRIVER_CURRENT_PATH}/wifi_pt.json)
+
+        find_package (Python3 REQUIRED COMPONENTS Interpreter)
+
+        # Wifi firmware blob
+        add_custom_target(${TARGET}_firmware_w_blob DEPENDS ${CMAKE_CURRENT_BINARY_DIR}/firmware_w_blob.S)
```

Comment:
It might be worth dropping the "w" only version for simplicity. I only added this in case there was an issue with the new "wb" firmware.

Reply:
Thanks - done